deployment/docker-compose/litellm_proxy_config.yaml

```diff
@@ -13,7 +13,7 @@ model_list:
       api_key: "os.environ/OPENAI_API_KEY"
   - model_name: detect-language
     litellm_params:
-      model: gpt-3.5-turbo-1106
+      model: gpt-4-0125-preview
```

Comment:
3.5 gest half wrong.
4 gets only two gibberish wrong as Unsupported

Reply:
Also update the AWS one: `deployment/aws/litellm_proxy/litellm_proxy_config.yaml`